ha/util/health_generator/mock_health_event_publisher.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+"""
+Module helps in generating the mock health event and publishing it to the
+message bus.
+
+"""
+
+import argparse
+import sys
+import pathlib
+
+from cortx.utils.conf_store import Conf
+from ha.util.conf_store import ConftStoreSearch
+
+docker_env_file = '/.dockerenv'
+_index = 'cortx'
+cluster_config_file = '/etc/cortx/cluster.conf'
+config_file_format = 'yaml'
+
+def is_container_env() -> bool:
+    """Returns True if environment is docker container else False."""
+    docker_env_file_path = pathlib.Path(docker_env_file)
+    if docker_env_file_path.exists():
+        return True
+    return False
+
+def get_data_nodes(conf_store: ConftStoreSearch = None) -> list:
+    """
+    Fetches data node ids using HA wrapper class and displays the result.
```

Comment:
I guess we do not display the result in this method.

Reply:
Not in this..But, its get returned and then there is print at the last which will take care of displaying the input.